pom.xml

```diff
@@ -104,7 +104,7 @@
         <version.org.jboss.jboss-dmr>1.4.0.Final</version.org.jboss.jboss-dmr>
         <version.org.jboss.jboss-vfs>3.2.12.Final</version.org.jboss.jboss-vfs>
         <version.org.jboss.logging.jboss-logging>3.3.1.Final</version.org.jboss.logging.jboss-logging>
-        <version.org.jboss.logging.jboss-logging-tools>2.0.1.Final</version.org.jboss.logging.jboss-logging-tools>
+        <version.org.jboss.logging.jboss-logging-tools>2.1.0.Alpha2</version.org.jboss.logging.jboss-logging-tools>
```

Comment:
@jamezp is going to look into this, whether the needed stuff can be in the 2.0.x stream or when the 2.1.x stream can get to .Final. It's too late to be bringing in alphas unless there is a clear plan to get them to .Final.